docs/source/commands/trtllm-bench.rst

```diff
@@ -0,0 +1,141 @@
+trtllm-bench
+===========================
+
+trtllm-bench is a comprehensive benchmarking tool for TensorRT-LLM engines. It provides three main subcommands for different benchmarking scenarios:
+
+**Common Options for All Commands:**
+
+**Usage:**
+
+.. click:: tensorrt_llm.commands.bench:main
+   :prog: trtllm-bench
+   :nested: full
+   :commands: throughput, latency, build
+
+
+Dataset Preparation
+------------------
+trtllm-bench is designed to work with the ``prepare_dataset.py`` script, which generates benchmark datasets in the required format. The prepare_dataset script supports:
```

Comment:
Maybe off-topic, is it possible to make prepare_dataset a subcommand for trtllm-bench? Like `trtllm-bench prepare_fake_dataset`, so that trtllm-bench can be an all-in-one toolkit.

Reply:
Yeah -- we have plans to eventually do that, but right now it's lower priority.